### ISSUE TITLE
[quesiton] How to create service?

### ISSUE THREAD

**Issue by xkwangcn** [redhat-developer/vscode-openshift-tools#2136]
When I use the Create Service command, it says 
<img width="457" alt="image" src="https://user-images.githubusercontent.com/29972855/117910948-785a5780-b30f-11eb-80ea-87444c456e5d.png">

But I can list catalog services in my cluster:
```
"/Users/wangxinke/.vscode/extensions/redhat.vscode-openshift-connector-0.2.5/out/tools/darwin/odo" catalog list services 

The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
wangxinkes-mbp:~ wangxinke$ "/Users/wangxinke/.vscode/extensions/redhat.vscode-openshift-connector-0.2.5/out/tools/darwin/odo" catalog list services 
Services available through Operators
NAME                                CRDs
etcdoperator.v0.9.4-clusterwide     EtcdCluster, EtcdBackup, EtcdRestore
service-binding-operator.v0.7.1     ServiceBinding

```

My question is how can I create service and use the link service function?

Thank you very much.

**Comment by dgolovin** [comment]
Extension currently supports only creating and linking services from service catalog. Operator backed services are not supported yet.

**Comment by xkwangcn** [comment]
@dgolovin how to create service catalog? I have installed Service Binding Operator (SBO).

**Comment by dgolovin** [comment]
@xkwangcn what version OpenShift/Kubernetes do you have?

**Comment by xkwangcn** [comment]
OpenShift 4.7

> @xkwangcn what version OpenShift/Kubernetes do you have?

**Comment by xkwangcn** [comment]
@dgolovin Hello, do you know how to create service catalog? I have installed Service Binding Operator (SBO). Thank you very much.

**Comment by dgolovin** [comment]
@xkwangcn unfortunately service-catalog is deprecated since OpenShift 4.3.x.

**Comment by dgolovin** [comment]
Fixed in #1787.
